docs/development/benchmark_with_djl.md

```diff
@@ -0,0 +1,217 @@
+# Benchmark your DL model with DJL
```

Comment:
Can shorten to "Benchmark your DL model". Considering this is in the DJL docs, users can guess they will be using DJL for it.